README.md

```diff
@@ -167,7 +166,6 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [remote-jobs](https://github.com/jessicard/remote-jobs) - A list of semi to fully remote-friendly companies in tech
   1. [Remotees](https://remotees.com/)
   1. [Remote.co Jobs](https://remote.co/remote-jobs/)
-  1. [RemoteList](https://remotelist.io/) - Has great job category filters

```

Comment:
seems to be working

Reply:
That's strange, just tried it again and can't get it to load.

Reply:
works only in FF. We can remove it then.